### PR TITLE
feat(billing.autorenew): add dialog before paying service while havin…

### PR DESCRIPTION
**Pull request by FredericEspiau**
## Add dialog before paying service while having a debt

Internal reference: SCFRCA-1317

### CODE REVIEW

File: client/app/account/billing/translations/Messages_fr_FR.xml [hunk 1]
```diff
@@ -62,6 +62,8 @@
    <translation id="autorenew_service_update_renewal_activation_notice" qtlid="369427">Le renouvellement automatique a été activé sur le(s) service(s) désigné(s) ainsi que votre compte.</translation>
    <translation id="autorenew_service_update_renewal_activation_error" qtlid="369440">Vous avez sélectionné le renouvellement automatique pour au moins un service mais nous n'avons pas pu activé le renouvellement automatique sur votre compte : {0}</translation>
    <translation id="autorenew_guide_help" qtlid="156732">Consultez nos guides en ligne.</translation>
+   <translation id="autorenew_service_debt_before_paying_1">Avant de vous engager sur une nouvelle période de temps, nous vous invitons à d'abord régler votre dette restante.</translation>
```
Comment (frenautvh) [inline client/app/account/billing/translations/Messages_fr_FR.xml]: i find this sentence is poorly written (no offense!) maybe replace with something like :
"Avant de vous engager sur une nouvelle durée, nous vous invitons au préalable à vous acquitter de votre dette."

File: client/app/account/billing/translations/Messages_fr_FR.xml [hunk 2]
```diff
@@ -62,6 +62,8 @@
...
+   <translation id="autorenew_service_debt_before_paying_2">Rendez-vous dans l’espace facturation afin de régulariser la facturation pour le service <strong>{{service}}</strong>.</translation>
```
Comment (frenautvh) [inline client/app/account/billing/translations/Messages_fr_FR.xml]: Same, facturation is redondant, maybe :
"Rendez-vous dans l’espace facturation afin de régulariser le service <strong>{{service}}</strong>."

**Comment by JeremyDec** [comment]
hello @frenautvh, as me and @FredericEspiau have same content to display, I can say that these phrases are not our responsibility. We have to set them like this. But yes it may sound weird :)

**Comment by frenautvh** [comment]
@JeremyDec so who wrote those sentences? i mean i know we're not writing a novel but come on ... it's so poorly written ...

**Pull request merged**